service/static/js-src/muk.publisher.js

```diff
@@ -4618,44 +4620,43 @@ $.extend(true, edges, {
                 var orderId = edges.css_id(namespace, "order", this);
                 var toggleId = edges.css_id(namespace, "toggle", this);
                 var resultsId = edges.css_id(namespace, "results", this);
+                var countClass = edges.css_classes(namespace, "count", this);
                 var results = "Loading...";
-                if (ts.values !== false) {
-                    results = "No data available"
-                }
-                if (ts.values && ts.values.length > 0) {
-                    results = "";
-                    var filterTerms = [];
-                    for (var i = 0; i < ts.filters.length; i++) {
-                        filterTerms.push(ts.filters[i].term.toString())
-                    }
-                    for (var i = 0; i < ts.values.length; i++) {
-                        var val = ts.values[i];
-                        if ($.inArray(val.term.toString(), filterTerms) === -1) {
-                            results += '<div class="' + resultClass + '"><a href="#" class="' + valClass + '" data-key="' + edges.escapeHtml(val.term) + '">' + edges.escapeHtml(val.display) + "</a> (" + val.count + ")</div>"
+                if (ts.values.length > 0) {
```

Comment:
false is the initial value...if something like the db connection going bad happens this value will not be changed otherwise the ts.values will return an array (filled or empty)